### ISSUE TITLE
Feature request =>  npm install --except X

### ISSUE THREAD

**Issue by ORESoftware** [npm/npm#16342]
#### I'm opening this issue because:

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [ ] npm is doing something I don't understand.
  - [x] Other (_see below for feature requests_):

#### What's going wrong?

I am running npm install on a couple of local repos (let's say 5 of them)

each repo has a fairly large devDependency X; I own X, and it's undergoing local development.

I am going to end up symlinking X into each of these 5 repos, so it actually doesn't make sense to install it at all, it's just a time suck as far as I can tell.

Is there a way to do

`npm install --except X`

? 

that would be really useful!

In other words, X would be an @latest entry for all 5 of these local repos. Why both install @latest from NPM, when in fact I want the absolute latest: the one on my filesystem!

**Comment by zkat** [comment]
There's changes to symlinks and local packages coming to npm@5 that might address this without `--except`.

**Comment by imerkle** [comment]
repoen this 

i cant run the `npm install` because it has a debian package and appveyour is windows.
therefore if i had `npm install --except  electron-installer-debian` there wouldnt be this problem
https://ci.appveyor.com/project/dsslimshaddy/monitor-gpu-el

**Comment by gilly3** [comment]
@dsslimshaddy that's what [`optionalDependencies`](https://docs.npmjs.com/files/package.json#optionaldependencies) are for.
